### PR TITLE
temporal assignment

### PR DESCRIPTION
**Pull request by gibsonMatt**
If possible, I would like to discuss the details of how to fit an ARIMA model. I am still pretty confused. 

Matt

### CODE REVIEW

**Comment by gibsonMatt** [comment]
added phyloTraits assignment to this pull request.

File: Week5-Temporal/temporal_assignment.Rmd [hunk 1]
```diff
@@ -80,16 +96,98 @@ In the R code chunk below, do the following:
...
+print(shannon <- diversity(siteBySpecies, index = "shannon"))
+
+siteShannon <- cbind(as.data.frame(matrix[,3]), as.data.frame(shannon))
+
+
+a <- aggregate(shannon ~ matrix[, 3], data=siteShannon, FUN=mean)
+op <- par(mar = c(10,4,4,2) + 0.1)
+
+barplot(a$shannon, names.arg=a$`matrix[, 3]`, las=2)
+
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: would be useful to see error bars

File: Week5-Temporal/temporal_assignment.Rmd [hunk 2]
```diff
@@ -80,16 +96,98 @@ In the R code chunk below, do the following:
...
 ```
 
 ***Question 2***: Describe how different biodiversity estimates vary among sites.
 
 a. Does diversity vary among sites? Does this correspond to treatment type?
 b. Is treatment type a significant predictor of site dissimilarity?
 
-> ***Answer 2a***: 
-> ***Answer 2b***:
+> ***Answer 2a***: Yes, diversity varies slightly among sites. There is a weak correspondance with treatment type as seen in the barplot. Long-term Krat exposure and Short-term Krat exposure plots had higher diversity on average. Rodent exclosures had the lowest diversity on average.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: PCoA explains good deal of variation, but biplots don't necessarily seem to capture this

File: Week5-Temporal/temporal_assignment.Rmd [hunk 3]
```diff
@@ -80,16 +96,98 @@ In the R code chunk below, do the following:
...
 
 ***Question 2***: Describe how different biodiversity estimates vary among sites.
 
 a. Does diversity vary among sites? Does this correspond to treatment type?
 b. Is treatment type a significant predictor of site dissimilarity?
 
-> ***Answer 2a***: 
-> ***Answer 2b***:
+> ***Answer 2a***: Yes, diversity varies slightly among sites. There is a weak correspondance with treatment type as seen in the barplot. Long-term Krat exposure and Short-term Krat exposure plots had higher diversity on average. Rodent exclosures had the lowest diversity on average.  
+> ***Answer 2b***:  Based on the PERMANOVA, treatment type is a significant predictor of site similarity (P = 0.001). More details on these effects can be seen in the indVal and Phi tests.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: elaborate?

File: Week5-Temporal/temporal_assignment.Rmd [hunk 4]
```diff
@@ -111,9 +265,9 @@ a. Does your data meet the assumption of stationarity? If not, what does this vi
 b. What does the ACF function do and how does it relate to the ARMA model? How does this differ from the autocorrelation function (ACF)?
 c. What results can you conclude from your full ARMA model along with other methods outlined in the time series setcion of the Handout?
 
-> ***Answer 3a***:
-> ***Answer 3b***:
-> ***Answer 3c***:
+> ***Answer 3a***: No, initially it did not. After the differencing correction, though, we can reject the null of non-stationarity (P = 0.0250). Not meeting the assumption of stationarity implies that the mean, variance, or covariance in this series is affected by time. 
+> ***Answer 3b***: I do not understand what two functions this question wants us to compare. The ACF function identifes the lags in our time series data. By looking at the correlations of lagged intervals, we can identify specific intervals that might be useful in parameterizing our final ARMA model.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: The ACF tells us about the lags of the forecast errors and thus provides information for the moving average (MA) component of the mode, while the PCF tells us about lags that can be addressed with the autoregressive (AR) component of the ARMA model. The differencing you did seemed to remove all significant lags in your time series, which should aid in fitting the ARMA model.

File: Week5-Temporal/temporal_assignment.Rmd [hunk 5]
```diff
@@ -111,9 +265,9 @@ a. Does your data meet the assumption of stationarity? If not, what does this vi
...
+> ***Answer 3c***: Based on the ARIMA model and predictions, we may expect rodent abundance (#/hectare) to fluctuate between ~225 and ~310 over the 20 years following 2002. I am not sure that I would put much faith in these projections, though. In the 22 years leading up to 2002, variance in rodent abundance from year to year was high. While the variance would ideally be captured in our ARIMA model, the residual random variation in the time series is quite high (see decomposition plots). I am also not entirely sure if I provided the correct parameters to the ARIMA function. I would like to have this discussed in more detail in class.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: fitting the "correct" ARIMA is not black and white. I think what may have happened here, however, is that it seems you ran the model with the non-differenced data, which may have affected your forecast. I wonder what the projections would have looked like had you run the ARIMA with abund.ts.diff

File: Week5-Temporal/temporal_assignment.Rmd [hunk 6]
```diff
@@ -148,14 +372,57 @@ In the R code chunk below, do the following:
...
 
 ```
 
 ***Question 5***:
 
 a. How does temporal turnover relate to spatial turnover?
-b. Which taxonomic group appears to be the most variable? Which group appears to be the least variable?
+b. Which taxonomic group appears to be the most variable? Which group appears to be the least variable?   
+    
+> ***Answer 5a***: The temporal turnover that we detect could be the result of spatial turnover occuring across the sampling area. In other words, the effects of spatial and temporal turnover are not entirely independent.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: not sure I entirely follow. The turnover metric is only using temporal data. I suppose once you average across replicates you are incorporating variation that occurs among different plots, but I'm not sure I would attribute that to "spatial".

File: Week5-Temporal/temporal_assignment.Rmd [hunk 7]
```diff
@@ -212,13 +653,17 @@ a. Which plot type has the highest stability in total abundance? How is stabilit
...
+> ***Answer 8b***: Synchony measures if and how species densities (for the same environment or treatment) are correlated over time. If they are strongly correlated, this may indicate that the species as a whole will respond to changes in the environment together and that they may be susceptible to environmental disturbances.
+> ***Answer 8c***: Regressing stability on mean richness verified the prediction of biodiversity-stability theory. Increased richness was associated with increased stability (R2 = 0.41). Conversely, when regressing stability on inverse of Simpson's diversity, we do not see this relationship. Although the plot looks like it has a slightly positive slope, R2 = -.039. Simpsons diversity takes into account richness AND evenness, so using this metric may provide a better indication of the true biodiversity-stability relationship.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: R2 tells us how much variation is explained by a model; seems strange that the value is negative.

File: Week5-Temporal/temporal_assignment.Rmd [hunk 8]
```diff
@@ -212,13 +653,17 @@ a. Which plot type has the highest stability in total abundance? How is stabilit
...
+A major challenge in studying spatial diversity is understanding the relative contributions of geographic distance and environmental divergence on species distributions. Often times these two factors are confounded. In other words, disentangling isolation-by-environment and isolation-by-distance to arrive at a formal conclusion on which is driving patterns of observed diversity is rather tough even if we can partition the variation.
+Two major challenges in studying temporal patterns of diversity is 1) accounting for non-independence of observations in longitudinal studies and 2) building acurate forecasting models to describe patterns of diversity into the future. The ARIMA model that I built did not do that great of a job capturing and forecasting the large amount of variance from year to year in rodent abundance. It is likely that the model significantly underestimated the variance. 
+
+
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: JTL: Nice job

**Pull request merged**